src/components/DataDecorator/DataDecorator.js

```diff
@@ -55,7 +55,21 @@ class DataDecorator extends Component {
       title,
       scoreThresholds,
       scoreDescription,
+      onClick: propOnClick,
+      onContextMenu: propOnContextMenu,
     } = this.props;
+    const onClick = !propOnClick
+      ? event => {
+          event.stopPropagation();
+          this.toggleOpen();
+        }
+      : propOnClick;
+    const onContextMenu = propOnContextMenu
+      ? event => {
+          event.preventDefault();
```

Comment:
Why is right-clicking being prevented? Is it being replaced with something?

Reply:
Yes! A custom secondary click menu is being created so the default browser one doesn't need to be displayed.